source/main/cardano/CardanoNode.js

```diff
@@ -225,6 +225,7 @@ export class CardanoNode {
     return Object.assign({}, this._status, {
       cardanoNodePID: get(this, '_node.pid', 0),
       cardanoWalletPID: get(this, '_node.wpid', 0),
+      rtsFlagsModeEnabled: isEqual(this._config.rtsFlags, RTS_FLAGS),
```

Comment:
rather than import another lodash function, you could use:
`this._config.rtsFlags.some(flag=> RTS_FLAGS.includes(flag))`?

Reply:
Well, I would say first use native, if not available you can use lodash. I, personally, prefer ramda (we need to discuss it). 

IMO `this._config.rtsFlags.some(flag=> RTS_FLAGS.includes(flag))` is not verbose enough nor justify to create a utils file for just one liner 😅